code/script.js

```diff
@@ -36,7 +122,7 @@ const showMessage = (message, sender) => {
 // Starts here
 const greetUser = () => {
```

Comment:
It could be an improvement for this first question to create a new button just for the name submission because right now it is a "submit soda info" button for the name submission.